pom.xml

```
@@ -47,8 +47,8 @@
 	</build>
 	
 	<scm>
-        <connection>scm:git:git://github.com/jenkinsci/hidden-parameter-plugin.git</connection>
-        <developerConnection>scm:git:git@github.com:jenkinsci/hidden-parameter-plugin.git</developerConnection>
+        <connection>scm:git:git://github.com/AleMartinez-Devops/hidden-parameter-plugin.git</connection>
+        <developerConnection>scm:git:git@github.com:AleMartinez-Devops/hidden-parameter-plugin.git</developerConnection>
```

The authoritative repository for the plugin should remain jenkinsci rather than AleMartiez-Devops.  The connection URL should switch from git protocol to https protocol because GitHub no longer support the unauthenticated git:// protocol.
```suggestion
        <connection>scm:git:https://github.com/jenkinsci/hidden-parameter-plugin.git</connection>
        <developerConnection>scm:git:git@github.com:jenkinsci/hidden-parameter-plugin.git</developerConnection>
```